_docs/tasks/installing-istio.md

```diff
@@ -17,86 +17,77 @@ This page shows how to install and configure Istio in a Kubernetes cluster.
+    * a text file containing the Istio version.
 
-3. Determine if your cluster has [RBAC enabled](https://kubernetes.io/docs/admin/authorization/rbac/) and find out the RBAC api version by running this command:
+3. Run the following command to determine if your cluster has [RBAC (Role-Based Access Control)](https://kubernetes.io/docs/admin/authorization/rbac/)
+enabled:
 
    ```bash
    kubectl api-versions | grep rbac
    ```
    * If the command displays an error, or does not display anything, it means the cluster does not support RBAC, and you can proceed to step 4.
-
-   * If the command displays 'alpha' version, please apply istio-rbac-alpha.yaml configuration:
+   * If the command displays 'beta' version, or both 'alpha' and 'beta', please apply istio-rbac-beta.yaml configuration:
    ```bash
-   kubectl apply -f istio-rbac-alpha.yaml
+   kubectl apply -f install/kubernetes/istio-rbac-beta.yaml
    ```
-    
-   * If the command displays 'beta' version, please apply istio-rbac-beta.yaml configuration:
+   * If the command displays only 'alpha' version, please apply istio-rbac-alpha.yaml configuration:
    ```bash
-   kubectl apply -f istio-rbac-beta.yaml
+   kubectl apply -f install/kubernetes/istio-rbac-alpha.yaml
    ```
 
-4. Install Istio's core components
-   (Istio-Manager, Mixer, Ingress-Controller, and optionally Istio CA for
-   authentication). There are two options at this stage:
+4. Install Istio's core components .
+   There are two mutually exclusive options at this stage:
 
-   **Without Istio Auth**:
+    * Install Istio without enabling [Istio Auth](https://istio.io/docs/concepts/network-and-auth/auth.html) feature:
 
    ```bash
-   kubectl apply -f istio.yaml
+   kubectl apply -f install/kubernetes/istio.yaml
    ```
+   This command will install Istio-Manager, Mixer, Ingress-Controller, Egress-Controller core components.
 
-   **With [Istio Auth](https://istio.io/docs/concepts/network-and-auth/auth.html)**
-   (For more information, please see [Istio Auth installation guide](./istio-auth.html)):
+   * Install Istio and enable [Istio Auth](https://istio.io/docs/concepts/network-and-auth/auth.html) feature:
 
    ```bash
-   kubectl apply -f istio-auth.yaml
+   kubectl apply -f install/kubernetes/istio-auth.yaml
    ```
 
-5. Source the Istio configuration file to set Istio environment variables:
-
-   ```bash
-   source ../../istio.VERSION
-   ```
+   This command will install Istio-Manager, Mixer, Ingress-Controller, and Egress-Controller, and the Istio CA (Certificate Authority).
 
-6. Install istioctl CLI. Use one of the [`istioctl`]({{home}}/docs/reference/commands/istioctl.html) client binaries corresponding to your OS: `istioctl/osx/istioctl`, `istioctl/windows/istioctl.exe`, `istioctl/linux/istioctl`, targeted at Mac, Windows or Linux users respectively. For example, run the following commands on a Mac system:
```

Comment:
There is no need to install istioctl like this anymore, it is contained in the downloaded installation files.